tests/exec-tests.sh

```diff
@@ -33,8 +33,8 @@ RUN sed -i 's|</dev/tty||g' /var/lib/githooks/install.sh && \\
     # Change the base template so we can pass in the hook name and accept flags
     sed -i -E 's|echo.*Hook not run inside a git repository.*|CURRENT_GIT_DIR=".git"|' /var/lib/githooks/base-template.sh && \\
     sed -i -E 's|GITHOOKS_RUNNER=(.*)|GITHOOKS_RUNNER=\1; GITHOOKS_RUNNER="\${GITHOOKS_RUNNER:-/var/lib/githooks/base-template.sh}"|' /var/lib/githooks/base-template-wrapper.sh && \\
```

Comment:
Maybe unrelated, but `GITHOOKS_RUNNER` does not seem to appear in `base-template.sh`, so this looks like a noop?

Reply:
Look closely its base-template-wrapper.sh :)

Reply:
Ah, indeed :-)

I was going to suggest to just set the global config, which of course works when running in docker / CI, but when running tests separately, you don't want to mess with a user's git config, so that won't really work.

But, this makes me wonder: Can't we just remove `--global` when reading config values, e.g. here: https://github.com/rycus86/githooks/pull/111/files#diff-13c7c6fb70b550a166546e1b5f5269a8R7

That would read the value from the current repo when it is set, or the global otherwise. That would allow setting the config value just for the current test repository, without needing this sed line. As an added bonus, it would also allow users to install multiple versions of githooks side-by-side and configure which one to use on a repo-by-repo basis by using non-global config values (not sure if that would be useful, but seems neat if it would be possible).

Reply:
Ok, thats right, we can remove --global, since this is better behavior I think!

Reply:
Done.

Reply:
Ok, so now the `--global` is gone, you could also remove this particular `sed` line (and then set the `githooks.runner` variable in each repository created). That's a bit of an invasive change, modifying all test steps AFAICS, so maybe that's better for later.

Reply:
Yeah, we leave that for another PR. We need also tests, maybe this goes hand in hand with your clone-set-and-use workflow...